content/en/metrics/custom_metrics.md

```diff
@@ -59,6 +59,9 @@ The following custom metric naming convention must be followed:
     {{< nextlink href="/developers/metrics/powershell_metrics_submission" >}}PowerShell{{< /nextlink >}}
     {{< nextlink href="/serverless/custom_metrics" >}}AWS Lambda{{< /nextlink >}}
     {{< nextlink href="/api/v1/metrics/#submit-metrics" >}}Datadog's HTTP API{{< /nextlink >}}
+    {{< nextlink href="/logs/log_configuration/logs_to_metrics/#generate-a-log-based-metric" >}}Generate Log-based metrics{{< /nextlink >}}
+    {{< nextlink href="/tracing/generate_metrics/#generate-span-based-metrics" >}}Generate Live Process-based metrics{{< /nextlink >}}
```

Comment:
```suggestion
    {{< nextlink href="/tracing/generate_metrics/" >}}Generate Live Process-based metrics{{< /nextlink >}}
```
For this one, I suggest we link to the top of the file, so people don't miss the beta banner